src/targets/gpu/device/include/migraphx/gpu/device/launch.hpp

```diff
@@ -73,7 +73,7 @@ MIGRAPHX_DEVICE_CONSTEXPR auto gs_invoke(F&& f, index_int i, index) -> decltype(
     return f(i);
 }
 
-inline auto gs_launch(hipStream_t stream, index_int n, index_int local = 1024)
+inline auto gs_launch(hipStream_t stream, index_int n, index_int local = 4096)
```

Comment:
is the 4096 applicable to our older hardware like vega10?

Reply:
I just ran rocminfo on my desktop, here is what I got:

```
  Wavefront Size:          64(0x40)                           
  Workgroup Max Size:      1024(0x400)  
```
So the max number of threads in a workgroup is 1024.